docs/concepts/consensus/overview.md

```diff
@@ -9,7 +9,10 @@ GoQuorum implements the following Proof of Authority consensus protocols:
 
 !!! warning
 
-    QBFT is currently an early access feature. It is not recommended for production networks with business critical impact.
+    - QBFT is currently an early access feature. It is not recommended for production networks with business critical impact.
```

Comment:
I understood that QBFT was ready, maybe we should remove this line. But please confirm with GoQuorum team.

Reply:
@antonydenyer ^

Reply:
I think this one is ready @NicolasMassart, @baptiste-b-pegasys raised #188 for this as well

Reply:
@antonydenyer - can we remove this line in GoQuorum now.  Lucas thought so but it's a GoQuorum question not a QBFT question.